.gitignore

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
Can you maybe add a `.gitignore` from [gitignore.io](https://www.gitignore.io/api/csharp%2Cvisualstudio) instead?

Reply:
Will do.